### DISCUSSION TITLE
Is it possible to share images?

### DISCUSSION THREAD

**Discussion by WoLfulus** [acorn-io/runtime#994] [General]
Lets say I have 3 containers, all of them would build from ./Dockerfile but runs in different ways.

Does acorn share the build between them or it will build them everytime?

How does that work and is it possible to add this feature if its not present yet?

Thanks!

**Comment by ibuildthecloud** [comment]
It will only build the image once.  If your docker image build has the same input (dockerfile, context dir, args, etc) acorn will detect that and only build the image once.
